django/db/backends/oracle/utils.py

```diff
@@ -40,3 +40,39 @@ def from_datetime(cls, dt):
             dt.year, dt.month, dt.day,
             dt.hour, dt.minute, dt.second, dt.microsecond,
         )
+
+
+class BulkInsertMapper:
+    types = {
+        'CharField': 'to_nchar(%s)',
```

Comment:
I'd tend to sort this list alphabetically rather than group by type and add some module constants for each string on the right side to prevent the repetition.